lego-mindstorms-pieces.py

```diff
@@ -90,6 +90,28 @@ def main():
     function(**kwargs)
 
 
+def load_new_element_ids():
+    """
+    load list of new element IDs
+    """
+    global newelementid_map
+    global newelementid_comment
```

Comment:
I cannot allow code with `global` to slip in. ([Why are global variables evil?](https://stackoverflow.com/questions/19158339/why-are-global-variables-evil#19158418))

Please create a class if you need to share values between functions.